state-chain/runtime/src/migrations/move_network_fees.rs

```diff
@@ -34,7 +34,7 @@ impl frame_support::traits::OnRuntimeUpgrade for NetworkFeesMigration {
 			>(b"CollectedNetworkFee");
 
 			// Bump the version of both pallets
-			StorageVersion::new(5).put::<pallet_cf_swapping::Pallet<Runtime>>();
+			StorageVersion::new(6).put::<pallet_cf_swapping::Pallet<Runtime>>();
```

Comment:
as above, this is unnecessary - will be fixed once rebased onto main after https://github.com/chainflip-io/chainflip-backend/pull/5090